Levi9.POS/Request/ProductRequest/ProductInsertRequest.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace Levi9.POS.WebApi.Request.ProductRequest
+{
+    public class ProductInsertRequest
```

Comment:
use DataAnnotations for validation

Reply:
I have added DataAnnotations for validation. 
[Required]
public string Name { get; set; }